NickvisionTagger.Shared/Models/PropertyMap.cs

```diff
@@ -154,4 +156,52 @@ public override string ToString()
         s += "=========";
         return s;
     }
+
+    /// <summary>
+    /// Check equlity of this PropertyMap with another one
+    /// </summary>
+    /// <param name="obj">PropertyMap to compare to as object</param>
+    /// <returns>True if PropertyMaps are equal, else false</returns>
+    public override bool Equals(object obj) => Equals(obj as PropertyMap);
+
+    /// <summary>
+    /// Check equlity of this PropertyMap with another one
+    /// </summary>
+    /// <param name="obj">PropertyMap to compare to</param>
+    /// <returns>True if PropertyMaps are equal, else false</returns>
+    public bool Equals(PropertyMap obj)
+    {
+        return obj.Filename == Filename &&
+            obj.Title == Title &&
+            obj.Artist == Artist &&
+            obj.Album == Album &&
+            obj.Year == Year &&
+            obj.Track == Track &&
+            obj.TrackTotal == TrackTotal &&
+            obj.AlbumArtist == AlbumArtist &&
+            obj.Genre == Genre &&
+            obj.Comment == Comment &&
+            obj.BeatsPerMinute == BeatsPerMinute &&
+            obj.Composer == Composer &&
+            obj.Description == Description &&
+            obj.Publisher == Publisher &&
+            obj.CustomProperties.Count == CustomProperties.Count && !obj.CustomProperties.Except(CustomProperties).Any();
+    }
+
+    public override int GetHashCode() => ToString().GetHashCode();
+
+    public static bool operator ==(PropertyMap obj1, PropertyMap obj2)
```

Comment:
- Add docs
- Make:
```csharp
public static bool operator ==(PropertyMap? obj1, PropertyMap? obj2) ///with ? since we have null
{
    if(obj1 is null || obj2 is null) //what if obj1 is not null but obj2 is...with your version, Equals will get called then crash as Equals needs a non-null PropertyMap since there is no ? next to it
    {
        return obj1 is null && obj2 is null;
    }
    return obj1.Equals(obj2)
}
```